src/plugins/intel_cpu/src/mkldnn_plugin.cpp

```diff
@@ -174,9 +174,16 @@ static void TransformationUpToCPUSpecificOpSet(const std::shared_ptr<ngraph::Fun
     const bool useLpt =
             _enableLPT &&
         ngraph::pass::low_precision::LowPrecision::isFunctionQuantized(nGraphFunc);
+    auto defaultPrecisions = ngraph::pass::low_precision::precision_set::int8_support;
+    bool hasINT16orINT32Levels = ngraph::pass::low_precision::LowPrecision::isFQLevelsPresent(
```

Comment:
Just initialize as `false` or `true` and call the function inside `if (useLpt)`.

Reply:
Didn't understand at first. Thanks for the clarification. It's done